nova_plugin/server.py

```diff
@@ -473,7 +473,14 @@ def get_server_by_context(nova_client):
 
 def _set_network_and_ip_runtime_properties(server):
     ips = {}
-    _, default_network_ips = server.networks.items()[0]
+    try:
```

Comment:
hi, why not:
if len(default_network_ips.items()) == 0 / if not default_network_ips.items():
    raise NonRecoverableError(...)
?